blockchain/validate_test.go

```diff
@@ -383,18 +388,14 @@ func TestSigOpsLimitsWithMultiMBBlocks(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Unexpected error creating sigOps test block: %v", err)
 	}
-	err = attachBigBlock(overBlockLimit)
-	isExpectedErr = false
+	err = validateBigBlock(overBlockLimit)
 	if err != nil {
 		if rule, ok := err.(RuleError); ok {
-			if rule.ErrorCode == ErrTooManySigOps {
-				isExpectedErr = true
+			if rule.ErrorCode != ErrTooManySigOps {
+				t.Fatalf("Expected to fail with TooManySigOps but got: %v", err)
 			}
```

Comment:
This has the same silencing issue. If the code ever swaps order so that `CheckTransactionInputs()` fails before the sigops check, this test would be a false pass.

Reply:
To make this explicit, would it be ok to add a comment here to this effect? Something like

// ignore errors after maxSigOps test